examples/private_information_retrieval/proto/private_information_retrieval.proto

```diff
@@ -0,0 +1,35 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+syntax = "proto3";
+
+package oak.examples.private_information_retrieval;
+
+import "google/protobuf/empty.proto";
+
+message Location {
+  float latitude = 1;
+  float longitude = 2;
+}
+
+message PointOfInterest {
+  string name = 1;
+  Location location = 2;
+}
+
+service PrivateInformationRetrieval {
+  rpc GetNearestPointOfInterest(Location) returns (PointOfInterest);
```

Comment:
It's good practice to define explicit request and response messages, see https://cloud.google.com/apis/design/naming_convention#request_and_response_messages . 

This is the perfect example: imagine we want to add a parameter to specify how many entries to return to the request, and return multiple entries in the response: in the current definition it would not be possible to do either of these things without making a breaking change.

Reply:
Done